docs/data-pipeline.md

```diff
@@ -12,7 +12,7 @@ The following terms are used throughout this document.
 - **Format Version:** A version of the file format, abstracted away from the schema version and data version.  For example, Protobuf 2 and Protobuf 3 are format versions.
 - **Format:** How the data is stored on disk or provided from a service.  Examples of data formats: JSON, YAML, Memory-Mapped, Protobuf.  The format is internal to the data provider.
 - **Hunk:** A small piece of locale data relating to a specific task.  For example, the name of January might be a hunk, and a list of all month names could also be considered a hunk.  A data piece is expected to reflect a specific type.
-- **LangID:** A tuple of language, script, and region.  LangID is a request variable.  Subtags should be handled by the ICU4X code rather than the data provider.
+- **LangID:** A tuple of language, script, and region.  LangID is a request variable.  Unicode Locale Extensions should be handled by the ICU4X code rather than the data provider.
```

Comment:
Just curious, was this change to help be more specific and narrow the scope to more immediately needs?

Reply:
I wanted to be more specific that the subtags I was talking about are the Unicode Locale Extensions.